packages/app/src/cli/commands/app/release.ts

```diff
@@ -0,0 +1,60 @@
+import {appFlags} from '../../flags.js'
+import {AppInterface} from '../../models/app/app.js'
+import {load as loadApp} from '../../models/app/loader.js'
+import Command from '../../utilities/app-command.js'
+import {loadExtensionsSpecifications} from '../../models/extensions/load-specifications.js'
+import {release} from '../../services/release.js'
+import {Flags} from '@oclif/core'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+import {addPublicMetadata} from '@shopify/cli-kit/node/metadata'
+
+export default class Release extends Command {
+  static description = 'Release an app version.'
+
+  static flags = {
+    ...globalFlags,
+    ...appFlags,
+    'api-key': Flags.string({
+      hidden: false,
+      description: 'The API key of your app.',
+      env: 'SHOPIFY_FLAG_APP_API_KEY',
+    }),
+    reset: Flags.boolean({
+      hidden: false,
+      description: 'Reset all your settings.',
+      env: 'SHOPIFY_FLAG_RESET',
+      default: false,
+    }),
+    version: Flags.string({
+      hidden: false,
+      description: 'Version tag of the version you want to release.',
+      env: 'SHOPIFY_FLAG_VERSION',
+      exclusive: ['app-version-id'],
+    }),
+    'app-version-id': Flags.string({
+      hidden: false,
+      description: 'Version identifier of the version you want to release.',
+      env: 'SHOPIFY_FLAG_VERSION_ID',
+      exclusive: ['version'],
+    }),
+  }
```

Comment:
I don't think we want to make this additional flag available. 

cc @jessehoek

Reply:
So I understand correctly: is this a `--app-version-id` flag? If so, I don't think we need this? We should just have a `--version` flag